lib/verify/response.rb

```diff
@@ -39,10 +39,10 @@ def scenario
     def identity_paramteters
       @identity_paramteters ||= {
         full_name: full_name,
-        address_line_1: address_lines[0],
-        address_line_2: address_lines[1],
-        address_line_3: address_lines[2],
-        postcode: address.fetch("postCode"),
+        address_line_1: address_line(1),
+        address_line_2: address_line(2),
+        address_line_3: address_line(3),
+        postcode: postcode,
```

Comment:
No address line 4?

Reply:
It's not clear how many address lines are returned from Verify. I'll add it in just in case it is returned